uui/components/overlays/DropdownMenu.tsx

```diff
@@ -116,7 +117,7 @@ export const DropdownMenuButton = React.forwardRef<any, IDropdownMenuItemProps>(
         return (
             <>
                 {isIconBefore && iconElement}
-                <Text cx={ css.caption }>{caption}</Text>
+                <Text cx={ props.indent ? [css.caption, css.indentMargin] : css.caption }>{caption}</Text>
```

Comment:
you can make it simpler: cx={ [css.caption, props.indent && css.indentMargin] }
Also let's rename 'indentMargin' class just to 'indent'